lib/commands/changelog.js

```diff
@@ -53,8 +53,9 @@ const showChangelog = co.wrap(function*(cwd, pkg, options) {
 
   const tasks = [ensureVersionTag, getPendingChanges, generateChangeLog];
 
-  for (const task of tasks) yield Promise.resolve(runTask(task));
+  for (const task of tasks) await Promise.resolve(runTask(task));
```

Comment:
```suggestion
  for (const task of tasks) await runTask(task);
```

`yield` bombed if it wasn't handed a Promise, but `await` doesn't mind.